.github/workflows/rust.yaml

```diff
@@ -10,7 +10,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Build
+    - name: Build no features 
       run: cargo build --verbose
-    - name: Run tests
```

Comment:
I understand the reasoning behind removing the tests here since they are already done in the code coverage action.  But in the code coverage action the tests are done with `--all-features`.  Maybe it's worth keeping a test job for each feature combination here?
- Test no features
- Test sync
- Test async_std
- Test async_std tokio_compat

Reply:
I only see that this is usefull to check & avoid _dependecies beetwen tests_ , because _dependencies between features_ has been solved in checked in #45. Is _dependencies between tests_ something that we need to care about? At first glance is hard to me to see a reason, but do you have anything in mind?